doc/configoptions.md

```diff
@@ -5,161 +5,102 @@ in `data/*/*.file_list` (see [file list docu](files.md)).
-- debug=ignore
+  perform a fix-up of update-alternatives links (because...? so that...?)
 
-  ignore noncritical errors
+- disjunct=WORD (eg. `initrd`, `root`)
 
-- debug=account
+  separate WORD from the rest (in what way? so that...?)
 
-  create disk usage statistics, stored in ```tmp/&lt;image_name&gt;.size```
+- filelist=FILELIST
 
-- dist=8.1, dist=8.1-i386+kde
-
-  build for this release
-
-- suserelease=8.0
-
-  obsolete, equivalent to dist=
-
-- kernel=&lt;kernel rpm&gt;
-
-  give kernel package name explicitly
+  use SRC/FILELIST.file_list
 
-- theme=&lt;theme&gt;
+- fs=FS (eg. `dir`, `squashfs`, `cpio`, default: `none`)
 
-  select this theme (default: theme=SuSE)
+  Use FS as the filesystem for the image.
+  May end with `.gz` or `.xz` for compression.
 
-- themes=&lt;list of themes&gt;
+- image=IMAGE (mandatory)
 
-  list of supported themes, support all if empty
+  Use tmp/IMAGE as temporary directory and image/IMAGE as the final image
 
-## root
+- tmpdir=f
 
-- uncompressed_root=1
 
-  build uncompressed image
+- kernel=KERNELRPM (eg. `kernel-default`)
 
-- keeproot=1
+  name of kernel RPM, available as `<kernel_rpm>` in *.file_list
 
-  don't rebuild everything, just recreate (copy the cached tree and compress) the
-  last image (useful to speed up testing)
+- libdeps=DEPS (a comma separated list)
 
-- filelist=foo
+  run `bin/check_libs` on the DEPS (in `tmp/`)
 
-  build using foo.file_list
+- mode=MODE (`add`, `keep`, default: ``)
 
-- nolibs=1
+  (the modes do something else than their names suggest)
 
-  no ldconfig check
+- nolinkcheck=1
 
-- imagetype=foo / imagetype=foo.gz / imagetype=none
+  don't check for dangling symlinks
 
-  use foo as filesystem for image (.gz: and compress with gzip);
-  none: don't create an image
+- nostrip=1
 
-- imagename=foo
+  ignored! bug? related to commit c109ec79
 
-  use tmp/foo & image/foo as temp dir & final image
+- src=SRC (eg. `boot`, `initrd`, `rescue`, `root+rescue`, default: IMAGE)
 
-- tmpdir=foo
+  look in the `data`/*SRC* directory.
 
-  use foo instead of tmp/root
+- tmpdir=*TMPDIR*
 
-## rescue
+  Use TMPDIR instead of tmp/root as the temporary directory.
 
-- keeprescue=1
+  The name is a lie. If it really were a temporary directory then the makefile
+  would not need to care to set it to specific values.
 
-  don't rebuild everything, just recreate (copy the cached tree and compress) the
-  last image (useful to speed up testing)
+<!-- old -->
 
-- filelist=foo
+- debug=DEBUG (comma separated tags, eg. `solv`, `filedeps`, `ignore`,
+  `ignorelibs`, `pkg`, `if`)
 
-  build using foo.file_list
+  `ignore`: ignore noncritical errors
 
-- imagetype=foo / imagetype=foo.gz / imagetype=none
-
-  use foo as filesystem for image (.gz: and compress with gzip)
-  none: don't create an image
-
-- imagename=foo
-
-  use tmp/foo & image/foo as temp dir & final image
-
-- tmpdir=foo
-
-  use foo instead of tmp/rescue
+- dist=8.1, dist=8.1-i386+kde
 
-## base
+  build for this release
```

Comment:
`dist` is outdated. At least I don't remember using it for a long time. The code around it very likely does not work in any useful way with current products.